internal/utils/comparators/configmaps/comparator.go

```diff
@@ -15,6 +16,9 @@ const (
 //AreSame compares configmaps for equality and return true equal otherwise false.  This comparison
 //only compares the data of the configmap by default unless otherwise configured
 func AreSame(actual *corev1.ConfigMap, desired *corev1.ConfigMap, options ...ComparisonOption) bool {
+	log.V(3).Info("Compare configmaps", "actual", actual)
+	log.V(3).Info("Compare configmaps", "desired", desired)
+	log.V(3).Info("Compare configmaps", "options", options)
```

Comment:
This seems very verbose. Does it make sense to remove it or increase the verbosity needed?